.storybook/preview.js

```diff
@@ -1,5 +1,15 @@
 import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
 import { Breakpoints } from '@/client/models/Style';
+import React from 'react';
+import { FocusStyleManager } from '@guardian/source-foundations';
+
+export const FocusManagerDecorator = (storyFn) => {
```

Comment:
might be worth adding the same explanatory comment that we have in `sourceAccessibility.tsx`